### ISSUE TITLE
Deprecated UIWebView API usage

### ISSUE THREAD

**Issue by tobynguyen** [braintree/braintree-ios-drop-in#193]
### General information

* SDK/Library version: 7.4.0
* Environment: Sandbox/Production
* iOS Version and Device: above iOS 12
* Integration type and version: CocoaPods 1.7.4

### Issue description

App Store Connect sends warning after uploading the app, that it contains a deprecated API usage of UIWebView. (ITMS-90809: Deprecated API Usage)

I don't know it's a false positive warning from the App Store side or what, but I keep getting this warning 2 weeks ago.

**Comment by sestevens** [comment]
Hi @tobynguyen. The Braintree SDK relies on a dependency called CardinalMobile that uses `UIWebView`. We should have a release of the Braintree SDK out soon that contains an updated version of CardinalMobile without `UIWebView`.

We'll post here once the release is out, and then you'll need to run `pod update` to get the latest version of the base Braintree SDK. You can also follow along with [this issue](https://github.com/braintree/braintree_ios/issues/481) in the base SDK repo.

**Comment by vlasov** [comment]
Hey @sestevens Braintree pod has been updated 3 days ago..
https://github.com/braintree/braintree_ios/issues/481#issuecomment-554534034

**Comment by sestevens** [comment]
Updating to the latest version of the base Braintree SDK (version 4.30.2) should resolve this issue for users of Braintree Drop-In. If you're using Cocoapods, you'll need to run `pod update`.

**Comment by vlasov** [comment]
@sestevens `pod update` will not have any effect, because Braintree Drop-In SDK is configured to  use particular version of Braintree SDK. In `BraintreeDropIn.podspec` file you can see:
```s.dependency "Braintree/Card", "~> 4.29"
    s.dependency "Braintree/Core", "~> 4.29"
    s.dependency "Braintree/UnionPay", "~> 4.29"
    s.dependency "Braintree/PaymentFlow", "~> 4.29"
    s.dependency "Braintree/PayPal", "~> 4.29"```
You have to update *podspec* and release a new version on BraintreeDropIn cocoapod in order to resolve this issue

**Comment by sestevens** [comment]
Hi @vlasov. The `~> 4.29` syntax pins to version 4.29 and higher, up to (but not including) version 5.0. Since the fix was released in version 4.30.2, `pod update` should allow you to update to this version.
